### PR TITLE
New commands for Jenkins

### PR DESCRIPTION
**Pull request by maxbeatty**
1. turn deploy jobs on and off - `@lennie turn off deploys`
2. change branches on playground - `@lennie put feature on playground1 for iris`
3. clean up playground by putting all branches back to `master` - `@lennie clean up playground`

### CODE REVIEW

File: scripts/jenkins-deploy.coffee [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    turnedOff = 0
+    requestsRemaining = 0
+
+    toggle = (url) ->
+      robot.http(url + cmd).post() (err, res, body) ->
+        if err
+          robot.emit 'error', e
+          msg.send 'Jenkins says: ' + err
+        else
+          turnedOff++
```
Comment (brianstarke) [inline scripts/jenkins-deploy.coffee]: Prefer ++turnedOff here

**Comment by maxbeatty** [comment]
@isocket/developers please review, ask questions, and :shipit: if you like

File: lib/util/jenkins.coffee [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      throw new Error 'process.env.HUBOT_JENKINS_URL needed for jenkins scripts'
+
+    jenkinsUrl = URL.parse process.env.HUBOT_JENKINS_URL
+
+    @client = robot.http(process.env.HUBOT_JENKINS_URL).auth(jenkinsUrl.auth)
+
+  _scope: (path, method, data, cb) ->
+    unless cb
+      cb = data
+      data = JENKINS_IS_STUPID
```
Comment (jeffharnois) [inline lib/util/jenkins.coffee]: why was this stored as a var and not just passed as an empty string?

Comment (maxbeatty) [inline lib/util/jenkins.coffee]: mostly to denote _why_ an empty string is being passed

**Pull request merged**